packages/office-ui-fabric-react/src/components/Popup/Popup.tsx

```diff
@@ -33,7 +33,11 @@ export class Popup extends React.Component<IPopupProps, IPopupState> {
 
   public componentDidMount(): void {
     if (this._root.current) {
-      this._disposables.push(on(this._root.current, 'focus', this._onFocus, true), on(this._root.current, 'blur', this._onBlur, true));
+      this._disposables.push(
+        on(this._root.current, 'focus', this._onFocus, true),
+        on(this._root.current, 'blur', this._onBlur, true),
+        on(window, 'keydown', this._onKeyDown as any)
```

Comment:
hmm. listening to window is a bad idea for 2 reasons;

1. you are listening to keypresses regardless of focus.
2. window is not what you think it is within child window scenarios. Use `getWindow(element)` to derive the correct window. Child window scenarios introduce a number of issues.

Reply:
ok, i can look into 2, but yes, i believe we need to listen to window. Normally, when the callout has focus, you can listen for key presses on the callout. But if it does not have focus, those keypresses aren't captured.

i.e. you can't use escape key to close callouts without focusable content

Reply:
getWindow updated. how do you feel about needing to listen for keydown's on the window when callout doesn't have focus?